[BUG]  Missing header file in USER_INTEL

**Summary**

Build exits with an error saying that "pppm.h" can't be opened.

**LAMMPS Version and Platform**

30Apr2019
Linux

**Actual Behavior**

```
In file included from /tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/src/USER-INTEL/verlet_lrt_intel.h(24),
                 from /tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/build/styles/style_integrate.h(4),
                 from /tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/src/lammps.cpp(27):
/tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/src/USER-INTEL/pppm_intel.h(30): catastrophic error: cannot open source file "pppm.h"
  #include "pppm.h"
                   ^

compilation aborted for /tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/src/lammps.cpp (code 4)
make[2]: *** [CMakeFiles/lmp.dir/build.make:2793: CMakeFiles/lmp.dir/tmp/makepkg/lammps-beta/src/lammps-patch_30Apr2019/src/lammps.cpp.o] Error 4
make[1]: *** [CMakeFiles/Makefile2:137: CMakeFiles/lmp.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

**Steps to Reproduce**

Compile with cmake flags: `-DINTEL_ARCH=cpu -DPKG_USER-INTEL=yes`

--- a/cmake/CMakeLists.txt
+++ b/cmake/CMakeLists.txt
@@ -1220,8 +1220,7 @@ if(PKG_USER-INTEL)
                          ${USER-INTEL_SOURCES_DIR}/fix_nh_intel.cpp
                          ${USER-INTEL_SOURCES_DIR}/intel_buffers.cpp
                          ${USER-INTEL_SOURCES_DIR}/nbin_intel.cpp
-                         ${USER-INTEL_SOURCES_DIR}/npair_intel.cpp
-                         ${USER-INTEL_SOURCES_DIR}/verlet_lrt_intel.cpp)
+                         ${USER-INTEL_SOURCES_DIR}/npair_intel.cpp)
 
   set_property(GLOBAL PROPERTY "USER-INTEL_SOURCES" "${USER-INTEL_SOURCES}")
 
@@ -1230,9 +1229,12 @@ if(PKG_USER-INTEL)
   RegisterNBinStyle(${USER-INTEL_SOURCES_DIR}/nbin_intel.h)
   RegisterNPairStyle(${USER-INTEL_SOURCES_DIR}/npair_intel.h)
   RegisterFixStyle(${USER-INTEL_SOURCES_DIR}/fix_intel.h)
-  RegisterIntegrateStyle(${USER-INTEL_SOURCES_DIR}/verlet_lrt_intel.h)
 
   get_property(USER-INTEL_SOURCES GLOBAL PROPERTY USER-INTEL_SOURCES)
+  if(PKG_KSPACE)
+    list(APPEND USER-INTEL_SOURCES ${USER-INTEL_SOURCES_DIR}/verlet_lrt_intel.cpp)
+    RegisterIntegrateStyle(${USER-INTEL_SOURCES_DIR}/verlet_lrt_intel.h)
+  endif()
 
   list(APPEND LIB_SOURCES ${USER-INTEL_SOURCES})
   include_directories(${USER-INTEL_SOURCES_DIR})

